practice_2/AnimalsApp/DetailsBreedModule/Presenter/DetailsBreedPresenter.swift

```diff
@@ -6,37 +6,55 @@ protocol DetailsBreedViewProtocol: AnyObject {
 }
 
 protocol DetailsBreedPresenterProtocol: AnyObject {
```

Comment:
What sholud I use here?
https://sarunw.com/posts/class-only-protocols-class-or-anyobject/
https://stackoverflow.com/questions/30176814/whats-the-difference-between-a-protocol-extended-from-anyobject-and-a-class-onl/32895975#32895975

Reply:
Answered in our school channel.
Good question though.
And it's really great if you ask such questions now. 
I believe you'll be a great Swift developer.

Reply:
Honestly, I still don`t understand what to use instead of AnyObject.
The keyword `class` is deprecated.
The official documentation says to use AnyObjeect "A class-only protocol is marked by its inheritance from AnyObject".

left the code as it is

Reply:
this comment is not a mandatory one.
just try to understand types system and use best practices as much as you can.